### DISCUSSION TITLE
v5 pgtype.Numeric to math/big.Rat

### DISCUSSION THREAD

**Discussion by cramey** [jackc/pgx#1450] [Q&A] [answered]
v4 supported Numeric.AssignTo() on math/big.Rat. v5 doesn't have AssignTo() and Scan() doesn't support big.Rat. Is there a method to convert a v5 pgtype.Numeric into big.Rat?

**Comment by jackc** [comment, marked as answer]
Not at the moment. It looks like that was added after v5 branched was branched and it was never ported. But in v5 the type system is more flexible so big.Rat can be supported without core support in the same way shopspring.Decimal is (e.g. https://github.com/jackc/pgx-shopspring-decimal).

**Reply by jay-babu** [reply]
Is there an existing repo that supports this already? Most people will likely be copying and pasting the same code to get it to work with Rat and Rat is part of the stdlib, so it is not an unusual feature to have centralized.
